build.rs

```diff
@@ -16,6 +32,29 @@ fn detect_fedora_feature() -> Result<()> {
     Ok(())
 }
 
+fn detect_el9_or_older() -> Result<()> {
```

Comment:
I'm not opposed to doing it this way but I think it will actually be cleaner to:

- Add a `fuse3` feature on the rust side
- Change the spec file to enable it

Look at how this is done for the RHSM stuff in the spec:
`%{?with_rhsm:--enable-featuresrs=rhsm}`

Ugh well...I guess we'd need to concatenate it in the spec file...

Reply:
Ahh I see, yeah that makes more sense.